artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/cursor/PagedReferenceImpl.java

```diff
@@ -74,6 +74,10 @@
 
    private long messageSize = -1;
 
+   private byte priority;
+
+   private boolean durable;
```

Comment:
need to default this not set somehow, possible use a byte to represent the boolean with 0 = false, 1 = true, -1 not set